app/ante_gate_test.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 const (
-	errCode     = uint32(19)
+	errCode     = uint32(19) // spam prevent respone code
```

Comment:
// spam prevention response code is a better comment here.

Reply:
fixed